test/ClientTest.php

```diff
@@ -0,0 +1,26 @@
+<?php /** @noinspection PhpUnhandledExceptionInspection */
```

Comment:
Completely unrelated but... any reason why amp isn't using declare strict types?

Reply:
Please don't hijack unrelated issues / PRs like that and open a new issue.